src/components/breadcrumbs.cjsx

```diff
@@ -30,12 +30,16 @@ module.exports = React.createClass
       if i is @props.currentStep
         classes.push('current')
         classes.push('active')
+        bsStyle = 'primary'
         # classes.push('disabled')
         title = "Current Step (#{step.type})"
 
       if step.is_completed
         classes.push('completed')
-        bsStyle = 'primary'
+        if step.correct_answer_id is step.answer_id or not step.correct_answer_id
+          classes.push('btn-success')
```

Comment:
No, I just use that class in the css to get the correct color, for the current step.  Bad naming, I'll update to reflect it as we go further.